src/tabs/presets/presets.js

```diff
@@ -208,12 +214,12 @@ TABS.presets.readDumpAll = function() {
     this.cliEngine.subscribeOnRowCame(str => {
         lastCliStringReceived = performance.now();
 
-        if (CliEngine.s_commandDiffAll !== str && CliEngine.s_commandSave !== str) {
+        if (CliEngine.s_commandDumpAll !== str && CliEngine.s_commandSave !== str) {
             diffAll.push(str);
         }
     });
 
-    this.cliEngine.sendLine(CliEngine.s_commandDiffAll);
+    this.cliEngine.sendLine(CliEngine.s_commandDumpAll);
```

Comment:
this should stay Diff not dump. The idea is to actually save diff as it's smaller, faster to save/load and more flexible.